app/controllers/UsersController.scala

```diff
@@ -168,13 +168,19 @@ class UsersController @Inject()(messagesApi: MessagesApi,
     )
   }
 
-  def login: Action[AnyContent] = action { implicit request =>
-    Ok(views.html.users.login(loginForm))
+  def login: Action[AnyContent] = action.async { implicit request =>
+    val username = configuration.get[String]("session.username")
+    val emailFromSession = EncryptionUtility.decrypt(request.session.get(username).getOrElse(""))
+    if (emailFromSession.isEmpty) {
+      Future.successful(Ok(views.html.users.login(loginForm)))
+    }
```

Comment:
Can we have else keyword on the same line as the closing curly braces? Something of the sort
```
if (..) {
.. 
} else {
..
}
```

Reply:
Yes, I have updated it and sending the PR